File: app/src/main/java/fergaral/datetophoto/activities/NoPhotosView.java
package fergaral.datetophoto.activities;

import android.content.Context;
import android.graphics.Canvas;
import android.graphics.Color;
import android.graphics.Paint;
import android.util.AttributeSet;
import android.view.View;

/**
 * Created by fer on 6/03/16.
 */
public class NoPhotosView extends View {

    private Paint paint;

    public NoPhotosView(Context context, AttributeSet attrs) {
        super(context, attrs);
        init();
    }

    private void init() {
        paint = new Paint(Paint.ANTI_ALIAS_FLAG);
        paint.setStyle(Paint.Style.FILL);
        paint.setColor(Color.GREEN);
    }

    @Override
    protected void onDraw(Canvas canvas) {
        super.onDraw(canvas);
        canvas.drawCircle(0, 0, Math.min(getMeasuredWidth() / 2, getMeasuredHeight() / 2), paint);
    }
}

File: app/src/main/java/fergaral/datetophoto/activities/NoPhotosView.kt
package fergaral.datetophoto.activities

import android.content.Context
import android.graphics.Canvas
import android.graphics.Color
import android.graphics.Paint
import android.util.AttributeSet
import android.view.View

/**
 * Created by fer on 6/03/16.
 */
class NoPhotosView(context: Context, attrs: AttributeSet) : View(context, attrs) {

    private var paint: Paint? = null

    init {
        init()
    }

    private fun init() {
        paint = Paint(Paint.ANTI_ALIAS_FLAG)
        paint!!.style = Paint.Style.FILL
        paint!!.color = Color.GREEN
    }

    override fun onDraw(canvas: Canvas) {
        super.onDraw(canvas)
        canvas.drawCircle(0f, 0f, Math.min(measuredWidth / 2, measuredHeight / 2).toFloat(), paint!!)
    }
}
